va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
In slice decoding base mode, VASliceParameterBufferBase instead of  VASliceParameterBufferH264 is required, HW will parse the header. 

In addition, pic_order_present_flag is used to pass  bottom_field_pic_order_in_frame_present_flag to the driver. FFmpeg has done it in this way:

    pps->cabac             = get_bits1(gb);
    pps->pic_order_present = get_bits1(gb);
    pps->slice_group_count = get_ue_golomb(gb) + 1;

    .pic_order_present_flag                 = pps->pic_order_present,